File: fizzbuzz.rb
def fizz_buzz(int)
  fizz_buzz_array = []

  (1..int).each do |i|

    if i % 3 == 0 && i % 5 == 0
      fizz_buzz_array << "FizzBuzz"
    elsif i % 3 == 0
      fizz_buzz_array << "Fizz"
    elsif i % 5 == 0
      fizz_buzz_array << "Buzz"
    else
      fizz_buzz_array << i.to_s
    end

  end

  fizz_buzz_array
end

File: javascript/fizzbuzz.js
fizzBuzz = (int) => {
  let fizzBuzzArray = []

  for(i = 1; i <= int; i++){
    if(i % 3 === 0 && i % 5 === 0){
      fizzBuzzArray.push("FizzBuzz")
    }
    else if(i % 3 === 0){
      fizzBuzzArray.push("Fizz")
    }
    else if(i % 5 === 0){
      fizzBuzzArray.push("Buzz")
    }
    else{
      fizzBuzzArray.push(i.toString())
    }
  }
  return fizzBuzzArray

}
// fizzBuzz(15)
